src/Rendering/AbstractBootstrapHorizontalRenderer.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types = 1);
+
+namespace Contributte\Forms\Rendering;
+
+use Nette\Forms\Rendering\DefaultFormRenderer;
+
+class AbstractBootstrapHorizontalRenderer extends DefaultFormRenderer
```

Comment:
abstract?

Reply:
it is common ancestor of both v3 and v4 horizontal renderers with common functionality (setting column widths). Or should I just duplicate code in both renderers?

Reply:
I ment `abstract class ..`.

Reply:
Ah, yes I see. I forgot that. I will fix it right away.